panel/applets/sensors/SensorsApplet.vala

```diff
@@ -0,0 +1,210 @@
+/*
+ * This file is part of budgie-desktop
+ *
+ * Copyright (C) 2014-2016 Mike Krüger <mikekrueger81@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ */
+
+public class SensorsPlugin : Budgie.Plugin, Peas.ExtensionBase
+{
+  public Budgie.Applet get_panel_widget(string uuid)
+  {
+          return new SensorsApplet(uuid);
+  }
+}
+
+public class SensorsApplet : Budgie.Applet
+{
+  protected Gtk.EventBox m_widget;
+  protected Gtk.Label m_label;
+  protected GLib.Menu m_menuMain;
+  protected string m_resultString = "Sensors";
+  protected bool m_running = false;
+  protected bool m_isCelsius = true;
+  protected Object m_lock = new Object();
+  protected bool m_exec;
+  protected Gtk.Popover ? m_popover = null;
+  protected Settings ? m_settings;
+
+  private unowned Budgie.PopoverManager ? m_manager = null;
+
+  public string uuid { public set ; public get; }
+
+  public SensorsApplet (string uuid)
+  {
+          Object(uuid: uuid);
+
+          settings_schema = "com.solus-project.sensors";
+          settings_prefix = "/com/solus-project/budgie-panel/instance/sensors";
+          m_settings = this.get_applet_settings(uuid);
+          m_settings.changed.connect(on_settings_change);
+
+          m_widget = new Gtk.EventBox();
+          m_label = new Gtk.Label(m_resultString);
+          m_widget.add(m_label);
+          m_label.show();
+
+          create_menuMain();
+          m_popover = new Gtk.Popover.from_model(m_widget, m_menuMain);
+          m_popover.get_child().show_all();
+
+          m_widget.button_press_event.connect((e)=> {
```

Comment:
Inconsistent styling.

 * 4 spaces
 * No space between words and `()`
 * Inconsistent function naming, switches between camel case and lowercase  splitting